From 4c36cf4fccc98407a53e90d9fdff38889a2beb08 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 2 Aug 2023 11:04:10 +0200
Subject: [PATCH 0414/1141] arm64: defconfig: disable PAGE_POOL_STATS in
 fragment-01-defconfig-cleanup

Page pool statistics track page allocation and recycling in page pools.
This option incurs additional CPU cost in allocation and recycle paths and
additional memory cost to store the statistics.

To avoid this CPU and memory costs, disable it, as statistics are not used.

Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Change-Id: I9e1cf8ead74d690c880c8e6f5486dd6d08aae5e9
---
 arch/arm64/configs/fragment-01-defconfig-cleanup.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-01-defconfig-cleanup.config b/arch/arm64/configs/fragment-01-defconfig-cleanup.config
index 9e39d7422c60..cf92bedd16c4 100644
--- a/arch/arm64/configs/fragment-01-defconfig-cleanup.config
+++ b/arch/arm64/configs/fragment-01-defconfig-cleanup.config
@@ -46,3 +46,4 @@
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 # CONFIG_MTD_PHYSMAP is not set
 # CONFIG_MTD_SPI_NOR_USE_4K_SECTORS is not set
+# CONFIG_PAGE_POOL_STATS is not set
-- 
2.39.2

